### PR TITLE
Rephrase the role of API Leader

### PR DESCRIPTION
**Pull request by timhoffm**
As proposed to the steering council the role should be broader than just
consistency, in particular it should cover API changes and it should
mediate between the concerns of usability, consistency and stability.

### CODE REVIEW

File: deputy_project_leads.md [hunk 1]
```diff
@@ -20,29 +20,29 @@ An individual may be the RM for more than one release series at the
...
-- deprecations are justified and properly documented
-- new functionality does not "paint us into a corner" for future work
-- new functionality is implemented with an API that is consistent with
-  the existing functions
+- additions are justified, i.e. they do not duplicate existing functionality
+  and do not exceed the intended scope of the library
+- additions are consistent with the existing API
+- additions are designed to not cause any future liabilities, i.e. they do not
+  unintendedly limit future extensions or expose internals
+- changes are carefully weighted between their benefit for future users and
```
Comment (dopplershift) [inline deputy_project_leads.md:35]: wording suggestion:

```suggestion
- changes are carefully balanced between their benefit for future users and
```

**Review by dopplershift — APPROVED** [review]
Sounds good to me. Minor wording suggestion, but I won't hold this up for that.

**Comment by dopplershift** [comment]
CI is failing because this is trying to build with Sphinx 5, which needs a new version of pydata-sphinx-theme.

**Comment by timhoffm** [comment]
Pinning sphinx<5 does not solve the issue. There's something broken in pydata-sphinx-theme :sad:

**Comment by dopplershift** [comment]
Could try pinning to `pydata-sphinx-theme!=0.9.0`

File: requirements-doc.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # List required packages in this file, one per line.
 mpl-sphinx-theme
```
Comment (timhoffm) [inline requirements-doc.txt:2]:
```suggestion
mpl-sphinx-theme<0.9
```

File: requirements-doc.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
 # List required packages in this file, one per line.
-mpl-sphinx-theme
+mpl-sphinx-theme<0.9
```
Comment (timhoffm) [inline requirements-doc.txt:2]:
```suggestion
mpl-sphinx-theme
pydata-sphinx-theme<0.9
```

**Review by dopplershift — APPROVED** [review]
Looks good.

**Review by efiring — APPROVED** [review]

**Pull request merged**